src/store.rs

```diff
@@ -114,6 +122,47 @@ impl LedgerStore for IndexerStore {
     }
 }
 
+impl CanonicityStore for IndexerStore {
```

Comment:
can we add canonicity to the BlockStore with something like `updateCanonicity`?

Reply:
I'm all for that. Would you minding doing it? I really need to focus on CI stuff

Reply:
got it bestie